pyuvdata/uvdata/ms.py

```diff
@@ -1943,11 +1964,11 @@ def read_ms(
         # importuvfits measurement sets store antenna names in the STATION column.
         # cotter measurement sets store antenna names in the NAME column, which is
         # inline with the MS definition doc. In that case all the station names are
-        # the same.
-        # TODO: Potentially flip which is the default -- use ant_names if filled and
-        # unique. This does seem to fail though on "day2_TDEM0003_10s_norx_1src_1spw".
-        # See the test "test_read_ms_read_uvfits".
-        if len(station_names) != len(np.unique(station_names)):
+        # the same. Default to using what the MS definition doc specifies, unless
+        # we read importuvfits in the history, of the antenna column is not filled
```

Comment:
I think there's a typo in this sentence. Maybe "history, of the antenna column" -> "history, or if the antenna column"?

Reply:
Ah, good catch -- fixed in 55fda3c.